monai/data/test_time_augmentation.py

```diff
@@ -89,7 +89,7 @@ def __init__(
         batch_size: int,
         num_workers: int,
         inferrer_fn: Callable,
-        device: Optional[Union[str, torch.device]] = "cuda" if torch.cuda.is_available() else "cpu",
+        device: Optional[Union[str, torch.device]] = None,
```

Comment:
I think maybe it's more clear to just use ` cpu` as the default value, `auto-select` logic is easy to cause unexpected behavior. You can refer to the ignite approach:
https://github.com/pytorch/ignite/blob/master/ignite/metrics/metric.py#L207
Which just use:
```py
device: Union[str, torch.device] = torch.device("cpu")
self.device = device
```
What do you think?
Others look good to me.

Thanks.